docs/curl/usertx.md

```diff
@@ -1,221 +1,417 @@
-				}
-			}
-		]
-	},
-    "signature": "aGVsbG8="
+          "user_id": "admin",
+          "tx_id": "7b6d6414-9b58-45d0-9723-1f31712add01",
+          "user_writes": [
+                {
+                      "user": {
+                            "id": "alice",
+                            "certificate": "MIIBsjCCAVigAwIBAgIRAJp7i/UhOnaawHTSdkzxR1QwCgYIKoZIzj0EAwIwHjEcMBoGA1UEAxMTQ2FyIHJlZ2lzdHJ5IFJvb3RDQTAeFw0yMTA2MTYxMTEzMjdaFw0yMjA2MTYxMTE4MjdaMCQxIjAgBgNVBAMTGUNhciByZWdpc3RyeSBDbGllbnQgYWxpY2UwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAASdCmAgHdqck7uhAK5siEF/O1EIUEIYtiR3XVEjbVhNe/6GXFShtsSThXYL9/XK6p4qF4oSy9j/PURMGnWbzSnso3EwbzAOBgNVHQ8BAf8EBAMCBaAwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMCMAwGA1UdEwEB/wQCMAAwHwYDVR0jBBgwFoAU7nVzp7gto++BPlj5KAF1IA62TNEwDwYDVR0RBAgwBocEfwAAATAKBggqhkjOPQQDAgNIADBFAiEAsRZlR4sDyxS//BJnYpC684EWu1hO/JU8rkNW6Nn0FFQCIH/p6m6ELkLNQpx+1QJsWWtH/LdW94WinVylhuA4jggQ",
+                            "privilege": {
+                                  "db_permission": {
+                                    "db1": 0,
+                                    "db2": 1
+                                  }
+                            }
+                      }
+                },
+                {
+                      "user": {
+                            "id": "bob",
+                            "certificate": "MIIBrzCCAVWgAwIBAgIQZOQpmvY31R8yeyy3ClrJtzAKBggqhkjOPQQDAjAeMRwwGgYDVQQDExNDYXIgcmVnaXN0cnkgUm9vdENBMB4XDTIxMDYxNjExMTMyN1oXDTIyMDYxNjExMTgyN1owIjEgMB4GA1UEAxMXQ2FyIHJlZ2lzdHJ5IENsaWVudCBib2IwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAASUDaIwGvRPPHHMzw4UFPTX5BTuPons8Xv3AR6k/8dDJQsn09qdtKWauLLLGxiLNDY2J8S0qPzJhJVPGF6h/l9Uo3EwbzAOBgNVHQ8BAf8EBAMCBaAwHQYDVR0lBBYwFAYIKwYBBQUHAwEGCCsGAQUFBwMCMAwGA1UdEwEB/wQCMAAwHwYDVR0jBBgwFoAU7nVzp7gto++BPlj5KAF1IA62TNEwDwYDVR0RBAgwBocEfwAAATAKBggqhkjOPQQDAgNIADBFAiAFxiyZgtiTwvMFF6jKtUE5vV0YzthpWmdRiUIbUclKzQIhALQolKPJl9xmv66wOyJTvR2q13Fb6j75M4WGcG4KfjDZ",
+                            "privilege": {
+                                  "db_permission": {
+                                        "db1": 0,
+                                        "db2": 0
+                                  }
+                            }
+                      },
+                      "acl": {
+                            "read_users": {
+                                  "admin": true
+                            }
+                      }
+                }
+
+          ]
+    },
+    "signature": "MEUCIHLCSwMzwxmnRfB6s1eON2bMfgDwFvxoSqaZ6ACXcbn0AiEA8KhjY56tSRg9Hh9UGchhGybTV2rWl1NcsAPLyW71Vu8="
 }'
 ```
+The user `alice` has read only access on the database `db1` and read-write access on the database `db2`. As the access control is not defined for the user,
+any user can read the credential and privilege of `alice` but only `admin` user can modify the properties of `alice` user. Similarly, the user `bob`
+has read only access on the database `db1`. Further, only the `admin` user can read the credential and privilege of `bob`.
```

Comment:
Explain that 
```
"db_permission": {
                                        "db1": 0,
                                        "db2": 0
                                  }
```
`0` is read, 
`1` is read-write

Reply:
Done. We should use `string` instead of `0` and `1`. The same is true for the validation codes. I have created an issue https://github.com/IBM-Blockchain/bcdb-server/issues/107